mylyn.context/org.eclipse.mylyn.team.ui/plugin.xml

```diff
@@ -17,14 +17,15 @@
    <extension-point id="commitTemplates" name="Commit Template Handlers" schema="schema/commitTemplates.exsd"/>
     
     <!-- needed to update change sets -->
+<!--
```

Comment:
Let's just remove this extension if we don't need it

Reply:
fixed with new commit